conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -11,5 +11,8 @@
 
 # Add all the application routes to the prod.routes file
 ->         /                          prod.Routes
-GET        /test-only/testOnlySecureMessaging                        @controllers.test.TestOnlySecureMessagingController.onPageLoad()
-POST       /test-only/testOnlySecureMessaging                        @controllers.test.TestOnlySecureMessagingController.onSubmit()
\ No newline at end of file
+GET        /test-only/createMessage                       @controllers.test.TestOnlySecureMessagingController.onPageLoad()
```

Comment:
can we adjust the test-only URLs to be like the other test-only URLs that include the standard service url path as well - else, this may cause conflicts with other services